jsk_perception/test/test_remove_blurred_frames.test

```diff
@@ -15,6 +15,8 @@
       timeout_1: 20
       topic_2: /head_camera/rgb/image_rect_color/laplacian_var
       timeout_2: 20
+      check_after_kill_node: true
```

Comment:
@iory 
I'm sorry, what is this test for?

Reply:
This test is for nodelet's destructor.
https://github.com/jsk-ros-pkg/jsk_recognition/pull/2681

Reply:
I see. This test fails in my environtment. Could you check please?

Reply:
Thank you!!